include/brod.hrl

```diff
@@ -17,104 +17,55 @@
 -ifndef(__BROD_HRL).
 -define(__BROD_HRL, true).
 
--type kafka_key()                 :: kpro:key().
```

Comment:
As in first comment, it's fine to forward the type definition to `brod:some_type()`.
The main concern here is that users should not experience compilation failure when the upgrade a semver minor.